### PR TITLE
[CI:DOCS]instructions for podman machine on macs

### PR DESCRIPTION
**Pull request by baude**
first pass for instructions on using podman machine on Intel and M1
macs.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: docs/tutorials/mac_experimental.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## SetUp
+
+You must obtain a compressed tarball that contains the following:
+* a qcow image
+* an app.entitlements file
```
Comment (jwhonce) [inline docs/tutorials/mac_experimental.md:7]:
```suggestion
* an entitlements.plist file
```

**Review by jwhonce — CHANGES_REQUESTED** [review]

File: docs/tutorials/mac_experimental.md (hunk 1 above)
Comment (baude) [reply docs/tutorials/mac_experimental.md:7]: actually i think this can be deleted now that we use the qemu one. agree?

**Review by jwhonce — APPROVED** [review]

**Comment by jwhonce** [comment]
LGTM

**Comment by jtligon** [comment]
very close
```INFO[0000] waiting for clients...                       
INFO[0000] listening tcp://0.0.0.0:7777                 
INFO[0000] new connection from  to /var/folders/_s/nx_bm4z94d97vpzh0tlgj20w0000gp/T/podman/qemu_podman-machine-default.sock 
Waiting for VM ...
qemu-system-aarch64: -accel hvf: invalid accelerator hvf
Error: dial unix /var/folders/_s/nx_bm4z94d97vpzh0tlgj20w0000gp/T/podman/podman-machine-default_ready.sock: connect: connection refused
ERRO[0003] cannot receive packets from , disconnecting: cannot read size from socket: EOF```

File: docs/tutorials/mac_experimental.md (hunk 1 above)
Comment (jtligon) [reply docs/tutorials/mac_experimental.md]: I didn't find a qemu one in the repo. so I made my own

File: docs/tutorials/mac_experimental.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+9. Sign all binaries
+
+   ```
+   sudo codesign --entitlements ~/qemu/accel/hvf/entitlements.plist --force -s - /usr/local/bin/qemu-* /usr/local/bin/gvproxy /usr/local/bin/podman
+   ```
+
+
+## Test podman
+
+1. run podman machine init --image-path /path/to/image
```
Comment (jtligon) [inline docs/tutorials/mac_experimental.md:80]: can we drop the "run" here and just turn these into commands to copy-paste?

Comment (rhatdan) [reply docs/tutorials/mac_experimental.md:80]: Yes

```
1.
podman machine init --image-path /path/to/image
```

File: docs/tutorials/mac_experimental.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+   ```
+   git clone https://github.com/qemu/qemu
+   ```
+
+5. Apply patches that have not been merged into upstream qemu.
+
+   ```
+   cd qemu
+   git config --global user.name "YOUR_NAME"
+   git config --global user.email johndoe@example.com
```
Comment (ashley-cui) [inline docs/tutorials/mac_experimental.md:40]: I'd be more comfortable with not using global so we don't mess with other git configs.
```suggestion
   git config user.name "YOUR_NAME"
   git config user.email johndoe@example.com

File: docs/tutorials/mac_experimental.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## SetUp
```
Comment (ashley-cui) [inline docs/tutorials/mac_experimental.md:3]:
```suggestion
## Setup
```

File: docs/tutorials/mac_experimental.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+You must obtain a compressed tarball that contains the following:
```
Comment (ashley-cui) [inline docs/tutorials/mac_experimental.md:5]: Probably a comment for later but we should let them know where to get the tarball from in step 8.

**Review by ashley-cui — COMMENTED** [review]
A few small things, but otherwise LGTM

File: docs/tutorials/mac_experimental.md (hunk 5 above)
Comment (baude) [reply docs/tutorials/mac_experimental.md:5]: well of course, but rn, i have no clue

File: docs/tutorials/mac_experimental.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+3. Uninstall the brew package
+
+   ```
+   brew uninstall qemu
+   ```
+
+4. Get upstream qemu source code.
+
+   ```
+   git clone https://github.com/qemu/qemu
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/mac_experimental.md:32]: Coming from a newby Mac user, how do I install git, curl, or other packages that are required but are not part of the default MacOS install?

Comment (TomSweeneyRedHat) [reply docs/tutorials/mac_experimental.md:32]: and where's the window that I run this from?

File: docs/tutorials/mac_experimental.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+   ```
+   mkdir build
+   cd build
+   ../configure --target-list=aarch64-softmmu --disable-gnutls
+   gmake -j8
+   sudo gmake install
+   ```
+
+
+8. Uncompress and place provided binaries into filesystem
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/mac_experimental.md:62]: "the filesystem"

File: docs/tutorials/mac_experimental.md (hunk 6 above)
Comment (jtligon) [reply docs/tutorials/mac_experimental.md:32]: Installing brew will pick up the Xcode cli tools with git and friends. You'll need the Terminal.app inside /Applications/Utilities/

**Pull request merged**